meta-mbl-distro/recipes-core/images/mbl-image-production.bb

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+###############################################################################
+# mbl-image-production.bb
+#   This file is the mbed linux OpenEmbedded recipe for building a minimal
+#   uboot/kernel/filesystem image only including production packages.
+###############################################################################
+inherit mbl-distro-image-production
+
+SUMMARY = "Mbed Linux Basic Minimal Image With Production Packages"
```

```suggestion
SUMMARY = "Mbed Linux OS basic minimal image with production packages"
```
... I suppose the existing `-development` recipe has the same issue.

---

Adressed in 816f1c0.